pkg/gen/node-templates/provider.ts.mustache

```diff
@@ -369,6 +368,17 @@ export namespace {{Group}} {
 
       {{/Properties}}
 
+      /**
+       * Get an existing {{Kind}} resource's state with the given name, ID, and optional extra
+       * properties used to qualify the lookup.
+       *
+       * @param name The _unique_ name of the resulting resource.
+       * @param id The _unique_ provider ID of the resource to lookup.
```

Comment:
Is "provider ID" the right term in the Kubernetes context?

Reply:
I thought this made sense to keep, but I can certainly make the comment more clear.